tools/shared.py

```diff
@@ -38,6 +38,9 @@
 EXPECTED_LLVM_VERSION = "13.0"
 PYTHON = sys.executable
 
+# Used only on Linux
```

Comment:
```suggestion
# Used only when EM_PYTHON_MULTIPROCESSING is set, see below
```

Reply:
I think this comment was missed? No big deal but would be good to update it eventually.

Reply:
(oh, it was because it had automerge and I approved it, heh...)